test/fixtures/output/network/list.golden

```diff
@@ -1,6 +1,6 @@
      ID    |         Name         | Cloud |   Region    |     CIDR      |             Zones              | Phase | Active Connection Types | DNS Resolution  
 -----------+----------------------+-------+-------------+---------------+--------------------------------+-------+-------------------------+-----------------
   n-abcde1 | prod-aws-us-east1    | AWS   | us-east-1   | 10.200.0.0/16 | use1-az1, use1-az2, use1-az3   | READY | TRANSITGATEWAY, PEERING |                 
+  n-abcde3 | prod-azure-eastus2   | AZURE | eastus2     | 10.0.0.0/16   | 1, 2, 3                        | READY | PRIVATELINK             | PRIVATE         
```

Comment:
We can add more test data here to ensure the sort is working as intended. 

A few examples to consider (and other permutations of these fields) : 
* Same `display_name`, different `cloud`, `region`, and `created_at`
* Same `display_name` and `cloud`, different `region, and `created_at`